### PR TITLE
Cancel previous running workflows

### PR DESCRIPTION
**Pull request by mzgubic**
I don't know anything about CI, so this might be very wrong - just testing it out

### CODE REVIEW

**Comment by devmotion** [comment]
I haven't used it anywhere (just opened a PR to KernelFunctions where we had an open issue about canceling workflows) but I think it could be helpful to configuration in https://github.com/styfle/cancel-workflow-action#advanced-pull-requests-from-forks such that it also works for PRs from forks.

**Comment by mzgubic** [comment]
Could we disable email notification when the workflow is cancelled rather than when it fails?

File: .github/workflows/Cancel.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+       - "JuliaNightly"
+     types:
+       - requested
+ jobs:
+   cancel:
+     runs-on: ubuntu-latest
+     steps:
+     - uses: styfle/cancel-workflow-action@0.9.0
+       with:
+         all_but_latest: true
```
Comment (mzgubic) [inline .github/workflows/Cancel.yml]: By default, only the workflows started before this action are cancelled. This flag cancels all but the latest workflow, which is helpful if the pipeline is saturated

**Comment by oxinabox** [comment]
Hmm this seems to have resulted in everything being cancelled because of VersionVigilante failing.
We don't want everything cancelled when anything fails, since Nightly fails all the time,

**Comment by mzgubic** [comment]
Oh no I cancelled manually because I didn't want to block other pipelines, sorry!

**Comment by oxinabox** [comment]
OK, lets give this a try

**Review by oxinabox — APPROVED** [review]

**Comment by mzgubic** [comment]
Did not seem to have worked, i.e. this PR still ran two full pipelines:
https://github.com/JuliaDiff/ChainRules.jl/pull/433

**Pull request merged**